### PR TITLE
update kfctl readme

### PR DESCRIPTION
**Pull request by QxiaoQ**
Delete '_--platform generatic_' flag from cmd '_kfctl.sh init_'

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/4092)

### CODE REVIEW

**Comment by QxiaoQ** [comment]
@jinchihe

**Comment by QxiaoQ** [comment]
Why I commited this code changes?

Because there is no "--platform generatic" option for kfctl 0.6.

**Comment by QxiaoQ** [comment]
@lluunn Could you please help take a look at this? Need approval to merge this changes.

File: kfctl/README.md [hunk 1]
```diff
@@ -13,14 +13,14 @@ be implemented in golang. The port to golang is because:
 
 The initial version of kfctl will seek parity with kfctl.sh by implementing the following subcommands:
 - `init`            Initialize a kubeflow application.
-- `generate`        Generate the k8 manifests of the kubeflow application.
-- `apply`           Submit the k8 manifests to the api-server
+- `generate`        Generate the k8s manifests of the kubeflow application.
+- `apply`           Submit the k8s manifests to the api-server
 - `delete`          Delete the kubeflow application
 
 Typical usage of `kfctl.sh` is as follows:
```
Comment (jlewi) [inline kfctl/README.md]: I think we deleted kfcth.sh so can you delete all the related instructions.

Comment (QxiaoQ) [reply kfctl/README.md]: @jlewi Thanks for your review, I will close this PR and create a new one to delete kfcth.sh related instructions.

**Pull request closed without merging**